jest.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import("jest").Config} */
+const config = {
+  preset: 'jest-expo',
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
+  setupFilesAfterEnv: ['@rnmapbox/maps/setup-jest'],
+  transformIgnorePatterns: [
+    'node_modules/(?!(...|@rnmapbox|(jest-)?react-native|@react-native(-community)?)|expo(nent)?|@expo(nent)?/.*|@expo-google-fonts/.*|react-navigation|@react-navigation/.*|@sentry/react-native|native-base|react-native-svg)',
+  ],
+  moduleNameMapper: {
+    // Jest seems to have issues with resolving '@comapeo/core-react' due to being a pure ESM module,
+    // but Metro doesn't when running the app
+    '^@comapeo/core-react': '<rootDir>/node_modules/@comapeo/core-react/dist',
+  },
```

Comment:
really annoyed with jest and its lack for proper ESM support. the documentation around it wasn't super helpful and trying to enable it led to other problems.

unfortunately, this was the only thing that worked so far. happy to field other ideas but otherwise, not sure if it's worth wasting time here or in the module to make things work because of jest limitations

Reply:
no longer relevant, as it's been solved by upgrading to `@comapeo/core-react@1.1.0` (see 047aab78827673b9d7af00a6e43bba9abf083ad9)